website/src/pages/en/token-api/faq.mdx

```diff
@@ -2,13 +2,13 @@
 title: Token API FAQ
 ---
 
-Get fast answers to easily integrate and scale with The Graph's high-performance Token API.
+Get fast answers to easily integrate and scale with The Graph's high-performance Token API. View the [OpenAPI document](https://token-api.thegraph.com/openapi) for endpoint formats and details.
```

Comment:
That seems like a strange place for this link. Maybe it could be in the answer to a question below, e.g. "do you follow the OpenAPI standard?"

Reply:
Ah, I see that [it's already there](https://github.com/graphprotocol/docs/pull/1015/files#diff-1cec4dd9a7e98b416b0a0d47aa05dcd9afcac2101b4a80bc1c084983aec44e99R43). I would simply remove it from here:
```suggestion
Get fast answers to easily integrate and scale with The Graph's high-performance Token API.
```